### DISCUSSION TITLE
Deleting reservations

### DISCUSSION THREAD

**Discussion by ghost** [alfio-event/alf.io#1272] [Q&A]
Hi! I'm Patricio, IT Manager of a students lead NGO from Argentina. We are really thankful for the tool you created as it's really useful for us.
I'm having some difficulties doing something and I was hoping I could get some help.
Sometimes, attendants make mistakes in their reservations or would like to cancel them. I'm struggling to find the cancellation button for a reservation that gets multiple tickets.
Thanks in advence

**Comment by cbellone** [comment]
Hi @patriciotourne-eo ,
thank you for using alf.io!
You can edit / cancel reservations from the backoffice. Here's how

1. Go to your Event's dashboard
2. click on the "Reservations" menu
![image](https://github.com/alfio-event/alf.io/assets/3385346/627a3caa-eac0-4824-ab33-25b5cedd11a7)
3. click on the Reservation you want to modify
4. Click on the "Cancel reservation" button if you want to cancel the entire reservation
![image](https://github.com/alfio-event/alf.io/assets/3385346/00f0ccc9-d864-4a01-b122-9d805052310e)
5. or scroll down to the attendee's list and remove one or more attendees from the reservation
![image](https://github.com/alfio-event/alf.io/assets/3385346/32002214-30b0-40b2-9d58-ff0e020b4326)

does this answer your question?

**Reply by ghost** [reply]
Yes!! That solves my problem. Thank you so much
